tests/test_reactive.py

```diff
@@ -347,7 +374,9 @@ def watch_show_ac(self, show_ac: bool) -> None:
 
     app = Calculator()
 
-    async with app.run_test() as pilot:
+    # Referencing the value calls compute
```

Comment:
If referencing calls compute methods, why do we need to call compute methods when other reactive variables are changed if those changes will only be observed when we reference the reactive attribute, which will then re-trigger a call to its compute method anyway?

Reply:
The main reason for `compute` method existing is that you can watch them, and have a `watch` method called when the result might change. Otherwise, you may as well define a property.

Reply:
Of course 🤦  `watch` methods are the key, here.